kustomize/commands/edit/remove/removesecret_test.go

```diff
@@ -92,16 +96,66 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 `,
 		},
+		"succeeds when one secret name exists in the specified namespace": {
+			input: `
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+secretGenerator:
+- name: test-secret
+  namespace: test-ns
+  files:
+  - application.properties
+`,
+			args: []string{"test-secret", "--namespace=test-ns"},
+			expectedOutput: `
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+`,
+		},
+		"handles empty namespace as default in the args": {
+			input: `
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+secretGenerator:
+- name: test-secret
+  namespace: default
+  files:
+  - application.properties
+`,
+			expectedOutput: `
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+`,
+			wantErr: false,
+			args:    []string{"test-secret"},
+		},
+		"handles empty namespace as default in the kustomization file": {
+			input: `
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+secretGenerator:
+- name: test-secret
+  files:
+  - application.properties
+`,
+			expectedOutput: `
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+`,
+			wantErr: false,
+			args:    []string{"test-secret", "--namespace=default"},
```

Comment:
Same question as the ConfigMap test scenarios.

Reply:
Added! I'll resolve this one and continue discussion on the other one.